docker-compose.dev.yml

```diff
@@ -152,6 +152,31 @@ services:
     environment:
       PORT: "8080"
       CI: "true"
+      DANGEROUSLY_DISABLE_HOST_CHECK: "true"
+    env_file:
+      - aleph.env
+
+  e2e:
+    build:
+      context: e2e
+    environment:
+      BASE_URL: "http://ui:8080"
+    volumes:
+      - "./e2e:/e2e"
+
+  worker:
```

Comment:
Added this as a more comfortable way of getting a worker up from a docker compose context.